src/_scss/components/checkbox/_filterCheckbox.scss

```diff
@@ -37,11 +37,17 @@
         display: flex;
         align-items: flex-start;
 
+        &:last-of-type {
+          margin-bottom: 0;
+        }
+
         input {
           @include usa-input-checkbox;
           margin: 0;
           border: solid rem(1) $gray-cool-20;
-          min-width: rem(14);
+          width: rem(12);
+          height: rem(12);
+          top: rem(3);
```

Comment:
there's already a top property on this checkbox from somewhere else, set to 2, and you're overwriting it here. can it be changed in the original file instead?